src/d_array.c

```diff
@@ -708,7 +708,7 @@ static t_int *tabsend_perform(t_int *w)
 {
     t_tabsend *x = (t_tabsend *)(w[1]);
     t_sample *in = (t_sample *)(w[2]);
-    int n = w[3];
+    t_int n = w[3];
```

Comment:
i think this is wrong.
`t_int` is practically never used for *int* values; it's main purpose seems to be to a way to provide "void-pointer arithmetic" in the dsp-graph.

(i think the use of `t_int` for `atom_getint` resp `atom_getintarg` is accidental and should be fixed)